GoMeet/src/main/webapp/js/vote-meeting-time.js

```diff
@@ -0,0 +1,178 @@
+let MAX_VOTES = 3; // Users can vote up to MAX_VOTES times.
+
+/**
+ * Function called on page load of meeting-event.html
+ * to fetch all relevant MeetingTime data from servlets 
+ * and generate voting form for the MeetingTimes.
+ */
+async function displayMeetingTimeForm() {
+  let meetingEventId = getMeetingEventId();
+  let meetingEvent = await MeetingEventDAO.fetchMeetingEvent(meetingEventId);
+  let meetingTimeIds = meetingEvent['meetingTimeIds']; 
+
+  if (('status' in meetingEvent && parseInt(meetingEvent.status) !== 200) 
+      || meetingTimeIds === null) {
+    // Execution cannot continue unless meetingTimeIds can be retrieved. 
+    // Must terminate script with an Error. 
+    throw new Error(generateErrorMessage(meetingEvent) + " - MeetingEventId: " + meetingEventId);
```

Comment:
This function throws exceptions and it's called from voteTime() but not wrapped in a try/catch block. What happens if this exception is thrown?

Reply:
There would be an error in the console _and_ the script execution will terminate, and from the user's perspective, the voting form section on the HTML page would be blank. 

Catching the error will allow execution to continue (but I want termination in this case, and it looks like throwing an error is the [typical way](https://stackoverflow.com/questions/9298839/is-it-possible-to-stop-javascript-execution) to terminate scripts). And once again, the poor user can't fix this issue, so I decided to not notify them via the UI or via `alert()` like I've done in the past.